### DISCUSSION TITLE
New Source: HDFS, Hive Integration

### DISCUSSION THREAD

**Discussion by jeesim2** [airbytehq/airbyte#35250] [New Connector Request]
## Tell us about the new integration you’d like to have
Which source and which destination? Which frequency?

* source
  * HIVE jdbc
  * hdfs (native)

## Describe the context around this new integration
Which team in your company wants this integration, what for? This helps us understand the use case.

* In many case we have to do `hdfs distcp` to move files from hdfs to another hdfs
* Sometimes source data can be collected through JDBC-hive from HIVE server

## Describe the alternative you are considering or using
What are you considering doing if you don’t have this integration through Airbyte?

* Apache Sqoop or `HDFS DFS command` to move files from hdfs to hdfs
* Build custom application doing `select query` through jdbc from hive server.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1200367912513076/1200368086422726) by [Unito](https://www.unito.io)

```[tasklist]
### Tasks
```

**Comment by ChristopheDuong** [comment]
Just curious about what is the ~size of the files that you are moving from/to HDFS in your use case?

**Comment by jeesim2** [comment]
@ChristopheDuong It depends. Sometimes around 1 GB. Tens of GB occasionally. : )

**Comment by yangtian9999** [comment]
Hi, did airbyte integrate the hive and hdfs already? Thanks.

**Comment by alaturqua** [comment]
Any updates on this?

**Comment by kpchen** [comment]
> Any updates on this?

+1 request

**Comment by quguangyu0321** [comment]
> > Any updates on this?
> 
> +1 request

+1 more request

**Comment by sherifnada** [comment]
hi everyone, could you please upvote the original issue? we can only see which tickets are most requested if you leave an 👍🏼 emoji reaction on the issue itself

**Comment by deenkar** [comment]
+1 more request For Hive Destination

**Comment by tomreitsma** [comment]
Would love to see HDFS integration!

**Comment by Gotodie** [comment]
+1 more request For Hive/Hdfs/HBase source

**Comment by sujiplr** [comment]
+1 more hive/hdfs target

**Comment by grishick** [comment]
Tagging this for destinations and sources teams until we split this issue into specific work items for specific connectors

**Comment by bryanpost** [comment]
Any update on this? Does the S3 connector work if a team is using EMRFS based out of S3? It doesn't appear so, but want to confirm.

**Comment by grishick** [comment]
S3 connectors (both source and destination) work with any S3 compatible endpoints. AFAIK, we have tested them with min.io

**Comment by bryanpost** [comment]
@grishick I'm sorry, my question was quite vague. My team uses Hive via EMRFS on S3, and I think [this issue re:partitions](https://github.com/airbytehq/airbyte/issues/6981) is more relevant. I need to do more research, but I don't think the existing S3 destination allows for creating the partitions required for Hive to work properly. Would be interested in hearing otherwise, however!

**Comment by chrismartin823** [comment]
> +1 more hive/hdfs target

+1 more request

**Comment by nikzasel** [comment]
> > +1 more hive/hdfs target
> 
> +1 more request

+1 more request)

**Comment by ext-herbert-santos** [comment]
+1 more request

**Comment by kissycn** [comment]
+1 more request For Hive/Hdfs/HBase source

**Comment by jonatasamorim** [comment]
We are still using Sqooq.
This connector would be really valuable.

**Comment by krutoileshii** [comment]
+1
